package.json

```diff
@@ -21,10 +21,13 @@
   "scripts": {
     "ci": "npm-run-all --continue-on-error --parallel test-cover lint",
     "lint": "eslint --max-warnings 0 .",
+    "lint-dockerfile": "docker run --rm -i hadolint/hadolint:latest-alpine < docker/Dockerfile",
     "lint-watch": "git ls-files | entr npm run lint",
     "test": "ava test/append-to-array-test.js test/markdownlint-cli2-test.js test/markdownlint-cli2-test-exec.js test/markdownlint-cli2-test-main.js test/merge-options-test.js test/resolve-and-require-test.js",
     "test-cover": "c8 --check-coverage --branches 100 --functions 100 --lines 100 --statements 100 npm test",
-    "test-watch": "git ls-files | entr npm run test"
+    "test-watch": "git ls-files | entr npm run test",
+    "build-container": "docker build -t davidanson/markdownlint-cli2:$SEMVAR -f docker/Dockerfile . && docker tag davidanson/markdownlint-cli2:$SEMVAR davidanson/markdownlint-cli2:latest",
+    "publish-container": "docker push davidanson/markdownlint-cli2:$SEMVAR && docker push davidanson/markdownlint-cli2:latest"
```

Comment:
I'm not sure I love SEMVAR. What about a Node one-liner to pull the version out of package.json instead? Something line `node -e "require('package.json').version"` (from memory on iPad)?

Reply:
I like it. I'll try that.

Reply:
got it working, it looks something like this:

```json
  "scripts": {
    "build-container": "VERSION=$(node -e \"process.stdout.write(require('./package.json').version)\") && docker build -t davidanson/markdownlint-cli2:$VERSION -f docker/Dockerfile . && docker tag davidanson/markdownlint-cli2:$VERSION davidanson/markdownlint-cli2:latest",
    "publish-container": "VERSION=$(node -e \"process.stdout.write(require('./package.json').version)\") && docker push davidanson/markdownlint-cli2:$VERSION && docker push davidanson/markdownlint-cli2:latest",
  },
```